/**:
  ros__parameters:
    map_resolution: 0.050
    kernel_scale: 8.0